pytest/tests/sanity/backward_compatible.py

```diff
@@ -31,6 +32,20 @@ def main():
         "--home=%s" % node_root, "testnet", "--v", "2", "--prefix", "test"
     ])
 
+    with open('/tmp/near/backward/test0/genesis.json') as f:
```

Comment:
How is this file generated?

Reply:
From above neard --home=node_root testnet

Reply:
that doesn't work because it still generates a config with current protocol version (the naming is confusing here). We should get the stable protocol version from s3 (or checkout the latest commit from the last stable version and get the protocol version from neard/src/genesis.json)

Reply:
No it's generated by `near-%s % stable_branch`, which is betanet binary